client/src/app/shared/shared-main/misc/top-menu-dropdown.component.html

```diff
@@ -4,26 +4,28 @@
     <a *ngIf="menuEntry.routerLink" [routerLink]="menuEntry.routerLink" routerLinkActive="active" class="title-page title-page-settings">{{ menuEntry.label }}</a>
 
     <div *ngIf="!menuEntry.routerLink" ngbDropdown class="parent-entry"
-      #dropdown="ngbDropdown" (mouseleave)="closeDropdownIfHovered(dropdown)">
+      #dropdown="ngbDropdown" autoClose="outside">
```

Comment:
When we click on an element, the dropdown is not automatically closed

Reply:
yes, I explained it also in the description : 
> Improve Dropdown : active class on active link + remove hover open + close outside click only
make sure the sub-menu is not too long horizontally - less confusing nav and more common

Reply:
Here it is for 2 things : 
- Accessibility, make sure the dropdown is still openened after click
- Avoid epileptic effect with the active class onclick (2 events trigged at the same time)

Reply:
@kimsible for the accessibility, how is this part of WCAG 2.1? for the epilepitc nature, it does not flash per WCAG 2.3.2.

Reply:
Invoking W3C spec would not help us here...but only crushing propositions with an authority argument.  
I don't see what you mean and I'm really opened to make any changes, maybe removing the active class and back to the autoClose=true (inside and outsite).

Reply:
> Invoking W3C spec would not help us here

I'm just trying to understand what hurdles we have to care about when it comes to designing the dropdown. If you just say "Accessibility", then I look up what corresponds to it. If you explain the reasoning behind classifying your proposition as more accessible, then we can compare.

Usually dropdowns are kept open for some elements (i.e. nested dropdowns), not just for navigation.

Reply:
> Accessibility, make sure the dropdown is still openened after click

I don't think users expect a menu to be still opened after a click. Every time we want to go to another page, we have to click 2 times.

Maybe a `menu`/`menubar` `role` could resolve the accessibility issue of dropdown auto close.

Reply:
> I don't think users expect a menu to be still opened after a click. Every time we want to go to another page, we have to click 2 times.
This PR goes back to the old effect https://github.com/Chocobozzz/PeerTube/pull/3023 but another PR should be created for the accessibility issue.